lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/address/api/handler/IssueCredentialHandlerTest.java

```diff
@@ -146,12 +149,11 @@ void shouldThrowJOSEExceptionWhenGenerateVerifiableCredentialIsMalformed()
         verify(mockEventProbe).counterMetric(ADDRESS_CREDENTIAL_ISSUER, 0d);
         verifyNoMoreInteractions(mockVerifiableCredentialService);
         verify(mockAuditService, never()).sendAuditEvent(any(AuditEventType.class));
-        Map responseBody = new ObjectMapper().readValue(response.getBody(), Map.class);
-        assertEquals(HttpStatusCode.BAD_REQUEST, response.getStatusCode());
-        assertEquals(ErrorResponse.VERIFIABLE_CREDENTIAL_ERROR.getCode(), responseBody.get("code"));
-        assertEquals(
-                ErrorResponse.VERIFIABLE_CREDENTIAL_ERROR.getMessage(),
-                responseBody.get("message"));
+        var responseBody = new ObjectMapper().readValue(response.getBody(), Map.class);
```

Comment:
Using Map.class here results in a "Raw use..." warning. We should be able to use:
Map<String, Object> responseBody =
                objectMapper.readValue(response.getBody(), new TypeReference<>() {});